databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -0,0 +1,484 @@
+               short_help='List shares.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_shares_cli(api_client):
+    """
+    List shares.
+
+    Calls the 'listShares' RPC endpoint of the Unity Catalog service.
+    Returns array of ShareInfos.
+
+    """
+    shares_json = UnityCatalogApi(api_client).list_shares()
+    click.echo(mc_pretty_format(shares_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get a share.')
+@click.option('--name', required=True,
+              help='Name of the share to get.')
+@click.option('--include-shared-data', default=True,
+              help='Whether to include shared data in the response.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_share_cli(api_client, name, include_shared_data):
+    """
+    Get a share.
+
+    Calls the 'getShare' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    share_json = UnityCatalogApi(api_client).get_share(name, include_shared_data)
+    click.echo(mc_pretty_format(share_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List permissions on a share.')
+@click.option('--name', required=True,
+              help='Name of the share to list permissions on.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_share_permissions_cli(api_client, name):
+    """
+    List permissions on a share.
+
+    Calls the 'getSharePermissions' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    perms_json = UnityCatalogApi(api_client).list_share_permissions(name)
+    click.echo(mc_pretty_format(perms_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update permissions on a share.')
+@click.option('--name', required=True,
+              help='Name of the share whose permissions are updated.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/unity-catalog/shares/{name}/permissions'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def update_share_permissions_cli(api_client, name, json_file, json):
+    """
+    List permissions on a share.
+
+    Calls the 'getSharePermissions' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).update_share_permissions(name, json))
+
+
+def shared_data_object(name):
+    return {'name': name, 'data_object_type': 'TABLE'}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update a share.')
+@click.option('--name', required=True,
+              help='Name of the share to update.')
+@click.option('--add-table', default=None, multiple=True,
```

Comment:
can we indicate in the help that this can be multiple values?

Reply:
Addressed in #471.